app/decorators/course_decorator.rb

```diff
@@ -46,6 +46,68 @@ def funding
     end
   end
 
+  def subject_name
+    if object.subjects.count == 1
+      object.subjects.first.subject_name
+    else
+      object.name
+    end
+  end
+
+  def has_scholarship_and_bursary?
+    has_bursary? && has_scholarship?
+  end
+
+  def bursary_first_line_ending
+    if requirements.count > 1
+      ":"
+    else
+      "#{requirements.first}."
+    end
+  end
+
+  def requirements
+    requirements = ["a degree of 2:2 or above in any subject"]
+    mathematics_requirement = "at least grade B in maths A-level (or an equivalent)"
```

Comment:
move to `if` block?

Reply:
`if` block is on line 73 - that what you mean?

Reply:
mathematics_requirement only used in line 74 right? move closer to usage?